src/less-style/structure.less

```diff
@@ -37,6 +37,23 @@ body.vellum-full-screen {
 
                 }
 
+                .fd-commander {
+                    display: flex;
+                    padding: 6px 4px 6px 12px;
+                    background-color: @cc-dark-cool-accent-mid;
+
+                    .fd-add-question-toggle {
+                        color: white;
+                        height: 17px;
+                    }
+
+                    input {
+                        border: 1px solid @cc-dark-cool-accent-low;
+                        flex: 2;
+                        margin: -2px 0;
```

Comment:
After reading a little more, realizing that you can use this to do things besides add questions also makes me want the commander to not look like it's part of the add question button. I do think it makes sense to be part of the question tree, though. Did you experiment at all with showing the commander above/below add question? I could imagine it being its own bar like the search bar in the add properties panel. Vertical space in the question tree is precious, but this is something that's easy to show/hide so I don't think that's a big deal.